applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -36,6 +36,8 @@ @interface AppLovinMAX()<MAAdDelegate, MAAdViewAdDelegate, MARewardedAdDelegate,
 @property (nonatomic, strong, nullable) NSArray<NSString *> *targetingKeywordsToSet;
 @property (nonatomic, strong, nullable) NSArray<NSString *> *targetingInterestsToSet;
 
+@property (nonatomic, strong) NSMutableDictionary<NSString *, NSString *> *extraParametersToSet;
```

Comment:
Same as Android.  Can you move this up to just below `locationCollectionEnabledToSet`?